src/models/PlayerStats.ts

```diff
@@ -207,9 +229,32 @@ serializer.addType({
   constructor: PlayerStats,
   schema: {
     type: 'object',
-    required: [ 'playerId', 'stats' ],
+    required: [ 'playerId', 'stats', 'ratings' ],
     properties: {
       playerId: { type:'string', format:'uuid' },
+      ratings: {
+        type: 'array',
+        subType: 'Map',
+        items: {
+          type: 'array',
+          items: [
+            { type:'string' },
+            {
+              type:'string',
+              oneOf: [
+                {
+                  required: ['rating'],
+                  properties: {
+                    rating: { type:'number' },
+                    lastUpdated: { type:'string', subType:'Date' },
```

Comment:
Nitpick.  The convention used elsewhere is to call the field "updatedAt".

Reply:
Fair enough, will update